features/modified_execution_paths/change_formatting_only_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative '../feature_helper'
+
+describe 'Changing and commiting a source file' do
+  include_context 'simple git repository'
+  include_context 'class1 examples'
+  include_context 'base forecast'
+
+  let(:strategies) do
+    [Crystalball::Predictor::ModifiedExecutionPaths.new]
+  end
+
+  it 'adds mapped examples to a prediction list' do
+    change class1_path, File.read(fixtures_path.join('class1_changed.rb'))
```

Comment:
what about example description? 

Also please rename fixture file to something more descriptive. E.g. `class1_formatting_changed.rb`